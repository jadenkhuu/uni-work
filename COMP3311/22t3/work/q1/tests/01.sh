psql racing -c 'select * from q1;'
