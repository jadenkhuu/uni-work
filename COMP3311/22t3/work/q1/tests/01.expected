   horse    
------------
 Brown Jack
(1 row)
